Description: Disable the OpenSSL engine support for the FreeBSD port.
Forwaded: not-needed
Author: Peter Pentchev <roam@FreeBSD.org>
Last-Update: 2010-09-20

--- src/ssl.c.orig
+++ src/ssl.c
@@ -256,6 +256,8 @@
 }
 
 static char *init_engine(void) {
+    s_log(LOG_ERR, "This version of stunnel was compiled WITHOUT support for OpenSSL hardware engines!  If you need this functionality, rebuild the FreeBSD port with the WITH_STUNNEL_SSL_ENGINE option set to 'yes'; contact Peter Pentchev <roam@FreeBSD.org> for details.");
+    exit(1);
     if(engine_initialized)
         return NULL; /* OK */
     engine_initialized=1;
